packages/react-native/src/primitives/Label/__tests__/__snapshots__/Label.spec.tsx.snap

```diff
@@ -1,39 +1,99 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Label applies style props 1`] = `
+exports[`Label applies accessibility role 1`] = `
+<Text
+  accessibilityRole="none"
+  style={
+    Array [
+      Object {
+        "color": "hsl(210, 50%, 10%)",
+        "fontSize": 16,
+        "fontWeight": "400",
+        "marginHorizontal": 4,
+        "marginVertical": 2,
+      },
+      Object {
+        "color": "hsl(210, 50%, 10%)",
```

Comment:
This is because we apply color to the `text` styles  and then also through variants. Since the default variant is primary we can remove color from `text` styles. What do you think about that?

Reply:
Makes sense to me 👍